### ISSUE TITLE
Theme Showcase: Inconsistent Premium Themes Labeling

### ISSUE THREAD

**Issue by Copons** [Automattic/wp-calypso#70080]
Premium themes are labeled inconsistently across Calypso and wp-admin:

| Plan | Calypso | wp-admin |
| - | - | - |
| Free | <img width="292" alt="Screenshot 2022-11-16 at 17 08 00" src="https://user-images.githubusercontent.com/2070010/202260344-ddb54bf3-d606-4fbe-a716-2c3216086f26.png"> | <img width="293" alt="Screenshot 2022-11-16 at 17 07 43" src="https://user-images.githubusercontent.com/2070010/202260374-d38b90db-4825-4aec-ad3c-276ffb8c7c57.png"> |
| Paid | <img width="296" alt="Screenshot 2022-11-16 at 17 09 19" src="https://user-images.githubusercontent.com/2070010/202260457-06840365-6ec6-496e-855c-f232e6f88252.png"> | <img width="294" alt="Screenshot 2022-11-16 at 17 09 29" src="https://user-images.githubusercontent.com/2070010/202260473-dcdbc9ef-ffee-4a29-8c8c-0a9efc263fbc.png"> |

Things that come to mind:

* The label should be consistent and as similar as possible.
* Can't help but question the Calypso ones. To me, they feel kind of the opposite of what they should be.
  * Do paid users (with premium themes included) need the loud green star?
    * What is its purpose, what is it signalling? Is it a way to teach users the value of the plan they are already paying?
    * With some searches, users would see the green star on most-if-not-all themes, which can be overwhelming.
  * Are free users able to notice the subtle grey star?
    * Is it intentional that this "upgrade nudge" is less visible where it should be more visible?
    * I'm aware that we use the star to indicate paid features, but I'm not sure if it's the right icon. The first thought I had when I saw it was that it was a bookmark button! _(which, btw, not the worst idea! 😄 )_
      * No idea if there is a commonly established icon for premium/paid features (a padlock? a dollar symbol?), but by and large, nowadays, `star` signifies: favorite, bookmark, like, etc.

cc @Automattic/serenity @SaxonF @ianstewart

**Comment by dsas** [comment]
> The first thought I had when I saw it was that it was a bookmark button! (which, btw, not the worst idea! 😄 )

It's a thing in Core! Maybe particularly useful for agencies / prolific site creators 

> No idea if there is a commonly established icon for premium/paid features (a padlock? a dollar symbol?
>
> I'm aware that we use the star to indicate paid features

We also use it to mean "Connect to stripe" on payment blocks 😅


![Image](https://user-images.githubusercontent.com/93301/202546693-518050f1-ae12-4f5f-b078-ba3635603ca8.png)

**Comment by Copons** [comment]
@dsas Yeah the star icon has been my pet peeve since the big GS Next Steps post (along with the wild usage of "Premium") 😄 

And, TIL about favoriting themes on Core!
I used to do it for plugins, but it never occurred to me that it was the same for themes. 😮
